aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,67 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[Use cases and example code](#when-to-use)  
+[Do's and Don'ts visual examples](#Dos)  
+[Use case and example code](use-cases)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sign up](#sign-up)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sign in](#sign-in)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Filter](#filter)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sort](#sort)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Change password](#change-password)   
+&nbsp;&nbsp;&nbsp;&nbsp;[Settings](#settings)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Pay](#pay)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Shipping](#shipping)   
+&nbsp;&nbsp;&nbsp;&nbsp;[Customize](#customize)  
+[Accessibility](#accessibility)   
+[Exceptions and how to address them](#exceptions-and-how-to-address-them)  
 
-Forms provide a way to collect important data from a user. However, this data is only collected effectively when Forms are built in a way in which users are inclined to fill out in full.
+### When to use
```

Comment:
```suggestion
## When to use
```

Reply:
@dahliaferrer - FYI, all the heading/# comments @taysea has left is regarding applying the [semantically correct headings](https://design-system.hpe.design/foundation/typography?q=semantic#semantic-usage-of-heading-levels) to the content. Let us know if you have questions.